GOVERNANCE.md

```diff
@@ -8,25 +10,84 @@ This document defines the project governance for Contour.
 
 ## Code Repositories
 
-The following code repositories are governed by Contour community and
-maintained under the `ProjectContour` namespace.
+All the code repositories owned by the Project Contour Github Organization are
+governed by Contour community and maintained under the `ProjectContour` namespace.
+The following repos are the most noteworthy:
 
 * **[Contour](https://github.com/ProjectContour/Contour):** Main Contour codebase.
-* **[Gimbal](https://github.com/ProjectContour/Gimbal):** Gimbal is an ingress load balancing platform capable of routing traffic to multiple Kubernetes and OpenStack clusters
 * **[community](https://github.com/ProjectContour/community):** Used to store community-related material–e.g., proposals, presentation slides, governance documents, community meeting minutes, etc.
-* **[ir2proxy ](https://github.com/ProjectContour/ir2proxy):** ir2proxy is a tool to convert Contour's IngressRoute resources to HTTPProxy resources.
-
+* **[contour-operator](https://github.com/ProjectContour/Contour):** The Contour Operator is a tool for installing and managing Contour.
 
 ## Community Roles
 
+All of the below roles share the responsibility to abide by the Code of Conduct and work together towards making the project better.
+
 * **Users:** Members that engage with the Contour community via any medium (Slack, GitHub, mailing lists, etc.).
 * **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
-* **Maintainers**: The Contour project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
+* **Reviewer:** Review contributions from other members.
+* **Maintainers**: The Contour project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. 
+* **SIG members/owner**: Members of specific SIG are reposnible for the SIG topic and are marked as maintainers 
+
+
+### User
+WIP: Needs definition
+
+### Contributor
+WIP: Needs definition
+### Reviewer
+WIP: Reviewers are able to review code for quality and correctness on some part of a subproject. They are knowledgeable about both the codebase and software engineering principles.
+
+#### Requirements
+
+* member for at least 3 months
+* Primary reviewer for at least 5 PRs to the codebase
+* Reviewed or merged at least 20 substantial PRs to the codebase
```

Comment:
This doesn't really work. They can't be the primary reviewer before they become a reviewer?